src/content/protocols/compound-v2.md

```diff
@@ -0,0 +1,338 @@
+| CErc20Delegate / CErc20 / CEther            | \_setInterestRateModel          | Accrues interest for the market and then set a new interest rate model using `_setInterestRateModelFresh`, pointing the market to a different rate‐calculation contract. A malicious admin could install a model that sets exorbitant rates or breaks interest accrual.                                                                                                                            | Timelock (DAO)                      |
++---------------------------------------------+---------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+
+| CErc20Delegate / CErc20 / CEther            | \_setInterestRateModelFresh     | Updates the interest rate model                                                                                                                                                                                                                                                                                                                                                                    | Timelock (DAO)                      |
++---------------------------------------------+---------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+
+| CErc20Delegate                              | \_becomeImplementation          | Permits admin to mark this contract as the new active implementation for its proxy. A malicious upgrade here could completely replace logic with a backdoor. permissions.                                                                                                                                                                                                                          | Timelock (DAO)                      |
```

Comment:
Looks unfinished. Could you explain more specifically what could a malicious update do